test/cypress/fixtures/statsBar.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "icon": "svguse:icons/slider_progress/lucide-route.svg#lucide-route",
+    "label": "udržitelných cest",
+    "value": "18"
+  },
+  {
+    "icon": "svguse:icons/slider_progress/jam-arrows-h.svg#jam-arrows-h",
+    "label": "",
+    "value": "312,25 km"
+  },
+  {
+    "icon": "svguse:icons/slider_progress/tabler-leaf.svg#tabler-leaf",
+    "label": "ušetřeno",
+    "value": "420 g CO2"
```

Comment:
Unit _gram CO2_ sholud be part of HTML, from the backend you get only number. Same for other unit cases.

Reply:
Updated labels based on `stat.id` key.